product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/02_virtual_network_peering.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Virtual Network Peering Example
```

Comment:
This page needs some caveats front and center about IP range conflicts. These are noted in the original Google Doc. /cc @zozowell